src/Host.cpp

```diff
@@ -559,7 +559,7 @@ const unsigned int Host::assemblePath()
     return totalWeight;
 }
 
-const bool Host::checkForMappingScript()
+bool Host::checkForMappingScript()
```

Comment:
Actually - is this possibly a `const` function - i.e. does not alter the *state* of the *class* it is in (or accesses)?

Reply:
I thought it would be but it is not, because it records state :)